### PR TITLE
Fix global

### PR DESCRIPTION
**Pull request by DanielRGonzalesV**

### CODE REVIEW

File: src/main/java/com/salesforce/dev/framework/utils/DataDrivenManager.java [hunk 1]
```diff
@@ -148,4 +149,25 @@ public DataDrivenManager(){
         }
         return chattersArray.iterator();
     }
-}
+
+    public Iterator<Object[]> getObjects(String nameJson, Class<?> elementClass) {
```
Comment (carledriss) [inline src/main/java/com/salesforce/dev/framework/utils/DataDrivenManager.java:153]: You need to apply this method over the duplicated methods.
so you need to remove getChatter, getCampaign, ....... etc
and also this method should be static

**Comment by carledriss** [comment]
@DanielRGonzalesV 
Please update you PR

**Comment by carledriss** [comment]
@SeleniumCourse/qadev07 
Any update of this task?

**Comment by HenrrySalinasJala** [comment]
@carledriss 
currently we(jala-fundaciontion) haven't internet access

**Pull request closed without merging**